src/Symfony/Bridge/Doctrine/Tests/Fixtures/User.php

```diff
@@ -58,6 +54,10 @@ public function getUserIdentifier(): string
         return $this->name;
     }
 
+    public function getSalt(): ?string
```

Comment:
this method is still needed? why move it inside the class?

Reply:
Bad rebase, now removed from the class.